### PR TITLE
Change changelog format

### PR DESCRIPTION
**Pull request by arnaudon**

### CODE REVIEW

**Comment by mgeplf** [comment]
Isn't the changelog included in the documentation?

**Comment by arnaudon** [comment]
its agonstic of format, no?

```

.. toctree::
   :hidden:

   readme
   cli
   changelog

```

**Comment by mgeplf** [comment]
AFAIK, rst looks for .rst files.
It looks like the docs have other issues, though.
For uniformity, I think it's best if we stay w/ rst.

**Comment by mgeplf** [comment]
> its agonstic of format, no?

It's also referenced here:
https://github.com/BlueBrain/atlas-direction-vectors/blob/main/doc/source/changelog.rst#L1

File: tox.ini [hunk 1]
```diff
@@ -66,3 +66,9 @@ commands =
 [pycodestyle]
 ignore = E203,E731,W503,W504,C0330,
 max-line-length = 100
+
+[gh-actions]
+python =
+  3.8: py38, lint
```
Comment (mgeplf) [inline tox.ini:72]: we already have this in the gh actions

File: tox.ini [hunk 2]
```diff
@@ -46,7 +46,7 @@ deps =
     sphinx
     sphinx-bluebrain-theme
     sphinx-click
-commands = make html SPHINXOPTS=-W
```
Comment (mgeplf) [inline tox.ini:49]: this is a useful check to make sure there aren't warnings, please don't remove it.

Comment (arnaudon) [reply tox.ini:49]: I'm testing atm, I couldn't fix the warning yet...

File: tox.ini (hunk 1 above)
Comment (arnaudon) [reply tox.ini:72]: where is it? I usualy put it here, but it does not work indeed, I wanted to add the docs test in the ci

Comment (arnaudon) [reply tox.ini]: found it!

File: tox.ini (hunk 2 above)
Comment (arnaudon) [reply tox.ini:49]: ah, it does not like the :nested: ok... but in the other similar file it works...

**Comment by mgeplf** [comment]
I pushed a version that seems to work, and goes back to the old style changelog.

**Pull request merged**